src/include/function/table_functions.h

```diff
@@ -49,9 +49,8 @@ struct TableFunctionInitInput {
     virtual ~TableFunctionInitInput() = default;
 };
 
-typedef std::unique_ptr<TableFuncBindData> (*table_func_bind_t)(main::ClientContext* /*context*/,
-    TableFuncBindInput* /*input*/, catalog::Catalog* /*catalog*/,
-    storage::StorageManager* /*storageManager*/);
+typedef std::unique_ptr<TableFuncBindData> (*table_func_bind_t)(
```

Comment:
We have a mix use of `typedef` and `using xxxx` statements in code. I suggest we replace all typedef statements with `using xxx` statements.

Reply:
I'll do this in a separate PR.